contrib/debian/copyright

```diff
@@ -2,13 +2,15 @@ Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: Bitcoin
 Upstream-Contact: Satoshi Nakamoto <satoshin@gmx.com>
  irc://#bitcoin-core-dev@libera.chat
-Source: https://github.com/bitcoin/bitcoin
+Source: https://github.com/vertcoin-project/vertcoin-core
 
 Files: *
-Copyright: 2009-2022, Bitcoin Core Developers
+Copyright:  2014-2022, Vertcoin Core Developers
+Copyright: 2009-2021, Bitcoin Core Developers
```

Comment:
Keep the Bitcoin Core on the same line (without modifying the year) and add Vertcoin underneath. Also there is one extra space before the years for Vertcoin